website/source/docs/provisioning/shell.html.md

```diff
@@ -129,6 +129,33 @@ Vagrant.configure("2") do |config|
 end
 ```
 
+In the code block above, the script block starts with `<<-SCRIPT` and ends with `SCRIPT`.
+This is known as a _Here Document_ or a _heredoc_.  Additionally, if your script
+relies on quotes and you do not wish for Ruby to escape your quotes, you may
+want to use this style of _heredoc_ where `SCRIPT` is surrounded in single quotes:
+
+```ruby
+$script = <<-'SCRIPT'
+echo "These are my \"quotes\"! I am provisioning my guest."
+date > /etc/vagrant_provisioned_at
+SCRIPT
+
+Vagrant.configure("2") do |config|
+  config.vm.provision "shell", inline: $script
+end
+```
+
+Now that our _heredoc_ is quoted, our script will preserve the quotes in the string to `echo`:
+
+```
+==> default: Running provisioner: shell...
+    default: Running: inline script
+    default: These are my "quotes"! I am provisioning my guest.
+```
+
+For more examples of how to use _heredoc_ in Ruby, please refer to the
```

Comment:
For more examples of how to use heredocs in Ruby, please refer to the

Reply:
Oops, missed these